src/components/Events/Events.js

```diff
@@ -6,10 +6,18 @@ import { EventsList, EventFilters, Loading } from '../';
 import { eventsAPI } from '../../api';
 import styles from './Events.sass';
 
+const odataFilterTypes = ['searchText', 'startDate'];
+const nonOdataFiltertypes = ['location', 'range'];
+
 function hasMoreEventsToLoad(currentPage, totalPages) {
   return currentPage + 1 < totalPages;
 }
 
+function buildFilterValues(filters, filterTypes) {
```

Comment:
I see what you mean, that makes sense.  If we can merge it now I can come back to it later when I have more time, but it would be handy for Devin et al to be able to play with it for now.